ext/tags.js

```diff
@@ -9,6 +9,9 @@ const tags = {
   SAMPLING_PRIORITY: 'sampling.priority',
   ANALYTICS: '_dd1.sr.eausr',
   ERROR: 'error',
+  ERROR_MSG: 'error.message',
```

Comment:
:memo: dd-trace-go changes `error.msg` to `error.messages` on 2022-12-07
https://github.com/DataDog/dd-trace-go/pull/1605

Reply:
What is your use case for setting these manually? It should be enough to set the `ERROR` tag to an error object which will then automatically be converted to the appropriate tags internally. Setting these manually is discouraged.